spec/features/check_pdf_download_link_spec.rb

```diff
@@ -43,19 +43,34 @@ def show
   end
 
   scenario "link will be disabled as first request will not be valid" do
-    given_i_am(:company01)
-    answer_all_questions
-    confirmation_of_supplied_details_page.submit_form
+    given_valid_data
+    start_a_new_et3_response
+    answer_respondents_details
+    answer_claimants_details
+    answer_earnings_and_benefits
+    answer_defend_claim_question
+    answer_representative
+    answer_disability_question
+    answer_employers_contract_claim
+    answer_additional_information
+    confirmation_of_supplied_details
 
-    expect(form_submission_page).to have_invalid_pdf_download
+    expect(form_submission_page).to have_valid_pdf_download
   end
 
   scenario "link will be enabled as second request will be valid" do
-    given_i_am(:company01)
-    answer_all_questions
-    confirmation_of_supplied_details_page.submit_form
+    given_valid_data
+    start_a_new_et3_response
+    answer_respondents_details
+    answer_claimants_details
+    answer_earnings_and_benefits
+    answer_defend_claim_question
+    answer_representative
+    answer_disability_question
+    answer_employers_contract_claim
+    answer_additional_information
+    confirmation_of_supplied_details
 
-    expect(form_submission_page).to have_invalid_pdf_download
     expect(form_submission_page).to have_valid_pdf_download
```

Comment:
Why has this changed ?  it seems to be nothing to do with the factory bot stuff.  My guess is that this test was valid in that it needs to see it be invalid and then change to valid

Reply:
Fixed in e324b47